### PR TITLE
More generic constraints

### PR DESCRIPTION
**Pull request by castrojo**
I ran through the readme and fixed up the documentation. I also made the constraints generic so we can run on GCE ootb.

Fixes issue #28

### CODE REVIEW

**Comment by mbruzek** [comment]
There was much debate on the topic of removing those constraints.

@marcoceppi did you want us to remove/comment out the constraints from this bundle?

File: bundle.yaml [hunk 1]
```diff
@@ -6,15 +12,15 @@ services:
     annotations:
       gui-x: '1200'
       gui-y: '600'
-    constraints: arch=amd64 cpu-cores=2 mem=4G root-disk=12G
+#    constraints: arch=amd64 cpu-cores=2 root-disk=12G
   kubernetes:
     charm: cs:~containers/trusty/kubernetes-5
     expose: true
```
Comment (mbruzek) [inline bundle.yaml]: You should include the release number to help resolve issue #30.

**Comment by mbruzek** [comment]
I would like to put in the version in this bundle and get a read on if removing constraints is acceptable before putting this through.

**Comment by castrojo** [comment]
I agree that testing a beefy bundle in CI is the right thing to do, but not at the expense of the bundle not working on public clouds ootb because we're asking for a ton of resources. 

cc @marcoceppi

**Comment by mbruzek** [comment]
I got advice that removing constraints was OK. +1

**Pull request merged**